contributing/testing-guide.md

```diff
@@ -163,4 +208,4 @@ Chromatic is set-up to check for regressions in all component stories. Typically
           └── [component-name]-test.stories.* (private stories)
 ```
 
-To introduce a new visual test, create a story in the relevant stories file that demonstrates the behaviour and it will automatically be detected by Chromatic.
+To introduce a new visual test, create a story in the relevant stories file that demonstrates the behaviour, and it will automatically be detected by Chromatic.
```

Comment:
Should we specify "once enabled" here? As in:
"To introduce a new visual test, create a story in the relevant stories file that demonstrates the behaviour, and, once enabled, it will automatically be detected by Chromatic."